01-messaging.md

```diff
@@ -108,6 +108,7 @@ The sending node:
   - MUST set feature bits as defined in [BOLT #9](09-features.md).
   - MUST set any undefined feature bits to 0.
   - SHOULD use the minimum lengths required to represent the feature fields.
+  - MUST set local feature `option_i_wumbo_you_wumbo` if `option_wumborama` is set in global features.
```

Comment:
this seems like an odd place to add this requirement. this restriction applies to the `wumbo` bit featureset, not to the `global/localfeatures` requirements. i'd move this to the `Requirements` section in features.